lib/Twig/Extensions/Extension/Text.php

```diff
@@ -22,6 +22,7 @@ public function getFilters()
         return array(
             new Twig_SimpleFilter('truncate', 'twig_truncate_filter', array('needs_environment' => true)),
             new Twig_SimpleFilter('wordwrap', 'twig_wordwrap_filter', array('needs_environment' => true)),
+            new Twig_SimpleFilter('repeat', 'twig_repeat_filter'),
```

Comment:
register ``str_repeat`` directly instead of wrapping it in a twig-specific function doing nothing else